include/hardening.mk

```
@@ -59,3 +60,8 @@ ifdef CONFIG_PKG_RELRO_FULL
   endif
 endif
 
+ifdef CONFIG_PKG_FANALYZER
+  ifeq ($(strip $(PKG_RELRO)),1)
+    TARGET_CFLAGS += -Wl,-z,now -Wl,-z,relro
```

This looks like a copy and past error:
```suggestion
    TARGET_CFLAGS +=  -fanalyzer
```